src/input_parsers/VnnLibParser.cpp

```diff
@@ -20,7 +20,7 @@
 #include "File.h"
 #include "InputParserError.h"
 
-#include <regex>
+#include <boost/regex.hpp>
```

Comment:
Turns out I could reproduce the CI error on my end with the debug build. 

It seems that std::regex is not very stable https://gcc.gnu.org/bugzilla/show_bug.cgi?id=61582. I tried to bump our C++ -std flag but that creates more issues. 

On the other hand using boost/regex seems to fix the issue on my end...